.ci/docker/common/install_onnx.sh

```diff
@@ -24,7 +28,7 @@ pip_install \
   transformers==4.25.1
 
 # TODO: change this when onnx-script is on testPypi
-pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@2bb3e9f2d094912f81cb63cecb412efb14c65738"
+pip_install "onnxscript@git+https://github.com/microsoft/onnxscript@2bb3e9f2d094912f81cb63cecb412efb14c65738" --no-deps
```

Comment:
fyi https://pypi.org/project/onnxscript-preview now available

Reply:
What is versioning for preview btw? is it like by dates like the nightly s?

Reply:
Yes. It's a two-times-a-week release version by dates